app/components/agenda/agenda-header/agenda-actions.js

```diff
@@ -306,7 +306,7 @@ export default class AgendaAgendaHeaderAgendaActions extends Component {
         if (job.status === job.SUCCESS) {
           const url = await fileDownloadUrlFromJob(job, name);
           debug(`Archive ready. Prompting for download now (${url})`);
-          fileDownloadToast.options.downloadLink = url;
+          fileDownloadToast.options.archiveLink = url;
```

Comment:
These changes won't work. This here is using the `Utils::Toaster::DownloadFileToast` component, which expects the field to be called `downloadLink`.